test/core/xds/xds_lb_policy_registry_test.cc

```diff
@@ -486,13 +486,13 @@ TEST(PickFirst, ShuffleOmitted) {
   EXPECT_EQ(*result, "{\"pick_first\":{\"shuffleAddressList\":false}}");
 }
 
-TEST(PickFirst, EnvVarDisabled) {
+TEST(PickFirst, EnvVarEnabled) {
```

Comment:
This test no longer has value, so let's just remove it.

Reply:
Done!